src/Services/SwaggerService.php

```diff
@@ -137,10 +137,13 @@ protected function generateEmptyData(): array
             throw new EmptyContactEmailException();
         }
 
+
+        $basePath = ltrim($this->config['basePath'], '/');
+        $basePath = blank($basePath) ? '' : '/'.$basePath;
         $data = [
             'openapi' => self::OPEN_API_VERSION,
             'servers' => [
-                ['url' => $this->getAppUrl() . $this->config['basePath']],
+                ['url' => $this->getAppUrl() . $basePath],
```

Comment:
seems it will works the same and will simplify code

```suggestion
                ['url' => URL::query($basePath)],
```

Reply:
Yes, It works👌